test/bigtest/tests/settings-test.js

```diff
@@ -0,0 +1,21 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import SettingsInteractor from '../interactors/settings';
+
+describe('Settings', () => {
+  const settings = new SettingsInteractor();
+
+  setupApplication();
+
+  describe('general', () => {
+    beforeEach(function () {
+      this.visit('/settings/oaipmh/general');
+    });
+
+    it('has a general settings message', () => {
+      expect(settings.generalMessage).to.equal('These are your general app settings.');
```

Comment:
Translations from en.json can be used there. Import the file here and use the needed field from it in order to avoid duplications of strings.

Reply:
fixed